modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -17,11 +14,6 @@ You can define labels and tags for each GCP resource only during {product-title}
 User-defined labels and tags are not supported for {product-title} clusters upgraded to {product-title} {product-version}.
 ====
 
-[NOTE]
```

Comment:
Please keep this note, it's still required.

Reply:
Ack